schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -88,6 +88,42 @@
       ],
       "description": "Scope indicates if the installer is per user or per machine"
     },
+    "NestedInstallerType": {
+      "type": [ "string", "null" ],
+      "enum": [

```

Comment:
Is it possible to have the enum values come from elsewhere?  It would be better to have the list defined once, then both nested and standard pull from it.

Reply:
I tried defining the enum values and reusing them for the NestedInstallerType. This will require using the `not` keyword which does not appear to be recognized by the library that converts the json schema to the C# model for winget-create. I will need to explicitly define the NestedInstallerType.